scanner/django.go

```diff
@@ -54,6 +54,18 @@ func configureDjango(sourceDir string, config *ScannerConfig) (*SourceInfo, erro
         vars["wsgiName"] = wsgiPath[0];
     }
 
+    settings, err := zglob.Glob(`./**/settings.py`)
+
+    if err == nil || len(settings) == 1 {
```

Comment:
Having one settings.py is the typical/basic case. 

What can happen here is having a module called `settings/` with files for each environment (base.py, staging.py, production.py, etc.) and all of those are considered "settings" files.

I'll check for both cases and also add a warning in case multiple files were found.

Reply:
@dangra is `s.DeployDocs` the only way to "warn" the users? 

All other languages/frameworks define only a general message on `s.DeployDocs` but I couldn't find any use of warning for specific checks. Any good practices you can suggest? 🙂

Reply:
honestly I don't know what is best practices for scanners. I will have to delegate that to @rubys if you don't mind. 

We can add something if DeployDocs doesn't suit or just reuse that until someone has time to look deeper into scanners

Reply:
Thanks! 

Yeah, I'm starting to have multiple info/warning messages to concatenate - and will have even more in the near future. 😁 For now, I can keep using DeployDocs but we can discuss about other ideas to display those info/warning/error messages and even differentiate them.